packages/fast-form-generator-react/src/form/form-item.select.tsx

```diff
@@ -70,6 +68,21 @@ class FormItemSelect extends FormItemBase<
         );
     }
 
+    /**
+     * Generates class names
+     */
+    protected generateClassNames(): string {
```

Comment:
can you be more specific on this method naming, for example `generateControlSpanClassNames`